README.md

```diff
@@ -12,7 +12,31 @@ and [BOSH][] manifests that allow provisioning of [CloudFoundry][] on AWS.
 
 ## Overview
 
-The following components needs to be deployed in order. They should be
+The high level process for deploying the PaaS environment is as follows. For a
+step by step guide, follow the more detailed instructions further down.
+
+1. Use the vagrant AWS plugin to deploy the "bootstrap-concourse" into your
+AWS environment. Once its deployed, vagrant will port forward access to the
+web UI for the bootstrap concourse to localhost over SSH. Use this to log into
+the bootstrap concourse UI from your web browser.
+
+2. Inside the bootstrap-concourse web UI, there are a number of 'pipelines' that
+you may run. Specifically, the `create-deplyer` pipeline is used that will then
+create a new "deployer-concourse" into that AWS environment. Once this is complete,
+you will be given the URL, username and password you can use to log into the
+'deployer concourse' web UI.
+
+3. Through the "deployet-concourse" Web UI, a number of new pipelines exist that
+you can run. Use the "create-bosh-cloudfoundry" pipeline to deploy BOSH and the
+cloudfoundry environments into the AWS environment.
+
+Note: Overnight, both the "bootstrap-concourse" and the cloudfoundry environment
+will auto delete itself to save unnecessary AWS costs. You will need to re running the
+"create-bosh-cloudfoundry" pipeline each day.
```

Comment:
Yes we shouldn't duplicate. Or just link to [Overnight deletion of environments](https://github.com/tobwilk/paas-cf/blob/readme-update/README.md#overnight-deletion-of-environments)